src/GovUk.Education.ExploreEducationStatistics.Notifier.Tests/Functions/ApiSubscriptionManagerTests.cs

```diff
@@ -0,0 +1,375 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Functions;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Functions;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Model;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Requests;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Notifier.Validators;
+using GovUk.Education.ExploreEducationStatistics.Notifier.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Notifier.Tests.Functions;
+
+public abstract class ApiSubscriptionManagerTests(NotifierFunctionsIntegrationTestFixture fixture)
+    : NotifierFunctionsIntegrationTest(fixture)
+{
+    private readonly Guid dataSetId = Guid.NewGuid();
+    private readonly string dataSetTitle = "data set title";
+    private readonly string email = "test@test.com";
```

Comment:
I think the names of these should be `_dataSetId`, `_dataSetTitle`, `_email`, which would help some with some warnings throughout this test class where these names are currently colliding with the names of method parameters.

Reply:
good shout